2018/_data/events.json

```diff
@@ -1,34 +1,5 @@
 {
   "items": [
-    {
-      "title": "Hands on performance workshop",
-      "url": "https://www.meetup.com/es-ES/BarcelonaJUG/events/240023918/",
-      "image": "/assets/img/events/performance.png",
-      "date": "May 25, 2017"
-    },
-    {
-      "title": "JSF with Primefaces, from ugly duckling to a beautiful swan",
-      "url": "https://www.meetup.com/BarcelonaJUG/events/238620028/",
-      "image": "/assets/img/events/primefaces.jpeg",
-      "date": "April 6, 2017"
-    },
-    {
-      "title": "Testing with Arquillian, Docker and Kubernetes",
-      "url": "https://www.meetup.com/BarcelonaJUG/events/238249460/",
-      "image": "/assets/img/events/testing-with-arquilian.jpg",
-      "date": "March 30, 2017"
-    },
-    {
-      "title": "The JVM as a platform for building smart contracts",
-      "url": "https://www.meetup.com/BarcelonaJUG/events/238293830/",
-      "image": "/assets/img/events/ben-evans.jpeg",
-      "date": "March 22, 2017"
-    },
-    {
-      "title": "Hands-on session: TDD in Baby Steps",
-      "url": "https://www.meetup.com/BarcelonaJUG/events/238052019/",
-      "image": "/assets/img/speakers/sandro-mancuso.jpg",
-      "date": "March 08, 2017"
-    }
+
```

Comment:
What about instead of removing all of them, include the ones that we already did this year?

Reply:
sure!